cmd/minikube/cmd/root.go

```diff
@@ -73,15 +74,24 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
-		shouldShowLibmachineLogs := viper.GetBool(showLibmachineLogs)
-		if glog.V(3) {
-			log.SetDebug(true)
+		if viper.GetBool(showLibmachineLogs) {
+			fmt.Println(`
+--show-libmachine-logs is deprecated.
```

Comment:
Suggestion: Should --show-libmachine logs still work as intended?  This will show a message to the user but will start minikube with this option not enabled which is what they desired.  Perhaps make it so this command sets logging level?  (I don't think this is necessary, just a suggestion)